def My_Func(x):
    assert x > 0, "Error"
    print(x)
print(My_Func(-1))

# has an error